src/auth/signup-form/signup-form.js

```diff
@@ -32,6 +32,19 @@ class SignUpForm extends Component {
             </Button>
           </div>
         </Form>
+        <div className="text-center text-muted signUp">Or sign up with</div>
+        <div className="text-center signUpButtons">
+          <Button type="submit" className="btn btn-primary">
```

Comment:
make these button part of form class maybe.

Reply:
Hi @vaibhavdaren  Did you mean that I should change the className sign to formSignup or something else?